sklearn/semi_supervised/label_propagation.py

```diff
@@ -32,7 +32,7 @@
 --------
 >>> from sklearn import datasets
 >>> from sklearn.semi_supervised import LabelPropagation
->>> label_prop_model = LabelPropagation()
+>>> label_prop_model = LabelPropagation(max_iter=1000)
```

Comment:
To be honest, I don't know. The `ConvergenceWarning` is partly there to allow users to adjust `max_iter` depending on their problem.